core/core.go

```diff
@@ -561,6 +562,23 @@ func runBazel(bazel string, args []string, out io.Writer) (int, error) {
 	return 0, nil
 }
 
+func isRetriableExitCode(exitCode int) bool {
+	// Bazel exits with 39 when remote cache evicts blobs, in which case, it can recover from error in next invocation.
+	return exitCode == 39
+}
+
+func runBazel(bazel string, args []string, out io.Writer) (int, error) {
+	retries := 0
+	for {
+		exitCode, err := runBazelOnce(bazel, args, out)
```

Comment:
We should probably print out some message notifying Bazelisk is retrying the bazel invocation?